contracts/gov/GovPool.sol

```diff
@@ -317,10 +335,57 @@ contract GovPool is
         descriptionURL = newDescriptionURL;
     }
 
+    function changeVerifier(address newVerifier) external override onlyThis {
+        verifier = newVerifier;
+    }
+
     function setNftMultiplierAddress(address nftMultiplierAddress) external override onlyThis {
         _setNftMultiplierAddress(nftMultiplierAddress);
     }
 
+    function saveOffchainResults(
+        bytes32[] calldata hashes,
+        bytes calldata signature
+    ) external override {
+        bytes32 signHash_ = getSignHash(hashes, block.chainid, address(this));
```

Comment:
Let's not pass the `block.chainid` and `address(this)` as arguments. The `getSignHash` function can hardcode them instead